### PR TITLE
allows to customize the mount path

### LINKED ISSUE #96: PodSecurityPolicy / VolumeMount need to be more configurable
**FEATURE REQUEST** 
context: 
In our case (CFCR a K8S deployed by bosh) the*.log files are not located in /var/log/containers but in /var/vcap/data/root_log/containers.
so the default volumeMounts are not revelant in our case. 
the podsecurity policy created, the allowed hostPast is a already defined list.

to solve that I think  there is several way to implement
either:
 just add a new extraVolumes with hostPath : /var/vcap/data/root_log/containers and extravolumemount   the podsecuritypolicy need also to add this new hostPath 

or maybe and easiest solution:
give the opportunity to use podsecuritypolicy but without create one (and so on the policy is created manually) and also default volumeMount need to be configurable

### PR DESCRIPTION
**Pull request by obeyler**
#### What this PR does / why we need it:
Every K8S distrib (such CFCR the K8S opensource by Cloudfoundry) doesn't store log at the same place as your default.
This PR allows to customize where the default mountPath are located and apply changment to the podsecuritypolicy according the new mountPath. The default value are the previous defined values.

#### Which issue this PR fixes
https://github.com/kiwigrid/helm-charts/issues/96
  - fixes #96

#### Special notes for your reviewer:
@monotek 
@axdotl 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://developercertificate.org) signed
- [X] Chart Version bumped (if the pr is an update to an existing chart)
- [X] Variables are documented in the README.md

### CODE REVIEW

File: charts/fluentd-elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: fluentd-elasticsearch
-version: 2.9.1
+version: 2.9.2
```
Comment (monotek) [inline charts/fluentd-elasticsearch/Chart.yaml]: new feature should raise minor version. -> 2.10.0

Comment (obeyler) [reply charts/fluentd-elasticsearch/Chart.yaml]: Ok I put 2.10.0

**Comment by obeyler** [comment]
@monotek It should be ok now :-)
@axdotl

**Pull request merged**